ruma-api/tests/conversions.rs

```diff
@@ -57,3 +59,85 @@ fn request_serde() -> Result<(), Box<dyn std::error::Error + 'static>> {
 
     Ok(())
 }
+
+#[test]
+fn request_with_user_id_serde() -> Result<(), Box<dyn std::error::Error + 'static>> {
```

Comment:
I don't like the verbosity of that error type. Since you only have one `?` below, why not change that to `unwrap()` instead?

Same for the other test function.

Reply:
Just copy&pasted from the `#[test]` above it to keep it consistent. But can change to `unwrap` if you prefer?